_posts/2019-02-27-stripe-connect-part-1.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: post
+title: "Implementing Stripe Connect in Rails | Part 1"
```

Comment:
What do you think of changing the title to `Implementing Stripe Connect in Rails: Part 1`. This is how it is in other articles we have.